app/components/articleSearch/components/searchItem/authors.tsx

```diff
@@ -1,30 +1,44 @@
 import * as React from "react";
 import { List } from "immutable";
-import { IAuthorRecord } from "../../../../model/author";
 import Tooltip from "../../../common/tooltip/tooltip";
+import { IPaperAuthorRecord } from "../../../../model/paper";
 const styles = require("./authors.scss");
 
 export interface IAuthorsProps {
-  authors: List<IAuthorRecord>;
+  authors: List<IPaperAuthorRecord>;
+}
+
+function getHIndexTooltip(hIndex: number) {
+  if (!!hIndex) {
```

Comment:
Be careful to use !! syntax to "number" type object.
If H-Index is 0, it could be false even though the zero actually means something.

Reply:
Thank you for your advice.
But I want to express like that, if H-index is 0, I think it is meaningless.